src/ParallelAlgorithms/Amr/Actions/Initialize.hpp

```diff
@@ -14,37 +13,31 @@
 #include "Utilities/MakeArray.hpp"
 #include "Utilities/TMPL.hpp"
 
-/// \cond
-template <size_t VolumeDim>
-class ElementId;
-/// \endcond
-
 namespace amr::Initialization {
-/*!
- * \brief %Mutator meant to be used with
- *  `Initialization::Actions::AddSimpleTags`to initialize flags used for
- * adaptive mesh refinement
- *
- * DataBox:
- * - Adds:
- *   - `amr::Tags::Flags<Dim>`
- *   - `amr::Tags::NeighborFlags<Dim>`
- * - Removes: nothing
- * - Modifies: nothing
- *
- */
+/// \ingroup InitializationGroup
+/// \brief Initialize items related to adaptive mesh refinement
+///
+/// \see InitializeItems
 template <size_t Dim>
 struct Initialize {
-  using return_tags =
-      tmpl::list<amr::Tags::Flags<Dim>, amr::Tags::NeighborFlags<Dim>>;
+  using const_global_cache_tags = tmpl::list<>;
+  using mutable_global_cache_tags = tmpl::list<>;
+  using simple_tags_from_options = tmpl::list<>;
+  using default_initialized_simple_tags =
+      tmpl::list<amr::Tags::NeighborFlags<Dim>>;
```

Comment:
just inline this in `simple_tags` below? As noted above, I think this isn't part of a generic interface.

Reply:
where do you want it documented that this adds a default initialized tag into the DataBox?

Reply:
that's part of the `simple_tags` interface, isn't it? That should be documented somewhere. I find the overlap between the `InitializeItems` interface and the `simple_tags` / `compute_tags` interface a bit confusing here, but nothing to fix in this PR.